src/mappings/entities/mapTrustedClaimIssuerTicker.ts

```diff
@@ -0,0 +1,33 @@
+import { Codec } from '@polkadot/types/types';
+import { SubstrateEvent } from '@subql/types';
+import { TrustedClaimIssuerTicker } from '../../types/models/TrustedClaimIssuerTicker';
+import { serializeTicker } from '../util';
+import { EventIdEnum, ModuleIdEnum } from './common';
+
+export async function mapTrustedClaimIssuerTicker(
+  _: number,
+  eventId: EventIdEnum,
+  moduleId: ModuleIdEnum,
+  params: Codec[],
+  __: SubstrateEvent
```

Comment:
(nit) Pretty annoying eslint doesn't handle this though.

```suggestion
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
  __: SubstrateEvent
```

?